data_structures/src/radon_error.rs

```diff
@@ -52,11 +52,17 @@ pub enum RadonErrors {
     TallyExecution = 0x53,
     /// Invalid reveal serialization (malformed reveals are converted to this value)
     MalformedReveal = 0x60,
+    /// Invalid data request serialization (malformed data requests are converted to this value)
```

Comment:
It is not true that malformed data requests are converted to this value.

```suggestion
    /// Requests that cannot be parsed must always get this error as their result.
    /// However, this is not a valid result in a Tally transaction, because invalid requests 
    /// are never included into blocks and therefore never get a Tally in response.
```